src/aurora/actionscript/object.cpp

```diff
@@ -53,6 +53,9 @@ std::vector<Common::UString> Object::getSlots() const {
 }
 
 bool Object::hasMember(const Common::UString &id) {
+	if (_members.find("constructor") != _members.end())
+		if (_members["constructor"].asObject()->hasMember(id))
```

Comment:
find() returns an iterator you can use directly, instead of having to go through operator[] again